### PR TITLE
refactor: cleanup `Box`

### LINKED ISSUE #7239: [Task]: Cleanup `Box`

### PR DESCRIPTION
**Pull request by evavirseda**
## Summary

Closes #7239 

## Changelog

```
- Cleanup Box
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/components/boxes/CopyableBox.svelte [hunk 1]
```diff
@@ -8,9 +8,7 @@
     export let value = ''
     export let isCopyable = true
     export let clearPadding = false
-    export let clearBoxPadding = false
     export let offset: number = undefined
-    export let classes = ''
```
Comment (cpl121) [inline packages/shared/components/boxes/CopyableBox.svelte:13]: When removing `classes` and `clearBoxPadding` from here, I think that in the `KeyValueBox` component it should also be removed
This change would also affect all sites where `KeyValueBox` is used 😅

**Review by cpl121 — CHANGES_REQUESTED** [review]

**Review by cpl121 — APPROVED** [review]

**Review by VmMad — CHANGES_REQUESTED** [review]
There seems to be changes in the following places:

The search box now looks weird
image
Original:
image

Same thing happens to amount input in the filters, and the change profile name in settings.

Tabs in node information (settings) looks weird too: (PR)
image
(Original)
image

And password inputs
image

**Review by VmMad — CHANGES_REQUESTED** [review]
just a few changes, so that the component is cleaned up a little more

**Review by VmMad — CHANGES_REQUESTED** [review]
for some reason the amount filter looks bigger now
image
image

**Review by VmMad — APPROVED** [review]
I think this is ready to go

**Pull request merged**